charts/tfy-karpenter-config/templates/karpenter-default-nodepool.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: karpenter.sh/v1beta1
+kind: NodePool
+metadata:
+  name: default
+  namespace: karpenter
+spec:
+{{ toYaml .Values.karpenter.defaultNodePoolSpec | indent 2 }}
```

Comment:
Let's move it to the same way we have for our other node pools.  Let's ensure consistency here with other nodepools